Samples/Samples.Android/MainActivity.cs

```diff
@@ -18,9 +19,14 @@ protected override void OnCreate(Bundle bundle)
             Xamarin.Essentials.Platform.Init(this, bundle);
             Xamarin.Forms.Forms.Init(this, bundle);
             Xamarin.Forms.FormsMaterial.Init(this, bundle);
+
+            Xamarin.Essentials.Platform.ActivityStateChanged += Platform_ActivityStateChanged;
```

Comment:
I think we need to unregister this as it will keep the activity alive. Although this is a single-activity app, we should unsubscribe for good measure.